.github/workflows/pr-integration.yml

```diff
@@ -133,3 +133,4 @@ jobs:
         name: wsm-logs
         path: |
           wsm.log
+          ${{ steps.integration-test.outputs.results_dir }}
```

Comment:
that seems nice :) but how do i get the log?

Reply:
I hope it will got to the Artifacts section on the run summary page, along with wsm.log (which is there now).

Reply:
i see! thanks :)

Reply:
If you want to test this pre-submit, you can just temporarily include a test fail in this PR